src/Microsoft.VisualStudio.Services.Agent/ConfigurationStore.cs

```diff
@@ -38,15 +38,14 @@ public sealed class AgentSettings
         [DataMember(EmitDefaultValue = false)]
         public string WorkFolder { get; set; }
 
-        // This flag is used to set if agent is configured as deployment agent or automation agent.
-        [DataMember(EmitDefaultValue = false)]
```

Comment:
I believe you don't use this anymore - you simply use the "MachineGroupId or DeploymentGroupId > 0" to know if this is Deployment Agent